Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace Promact.Trappist.Test.Tests
+{
+    [Collection("Register Dependency")]
+    public class TestsRepositoryTest
+    {
+        private readonly Bootstrap _bootstrap;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the '_bootstrap' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)